src/main/frontend/fs/capacitor_fs.cljs

```diff
@@ -236,18 +291,8 @@
          content)
        (p/catch (fn [error]
                   (log/error :read-file-failed error))))))
-  (delete-file! [_this repo dir path {:keys [ok-handler error-handler]}]
-    (let [path (get-file-path dir path)]
-      (p/catch
-       (p/let [result (.deleteFile Filesystem
-                                   (clj->js
-                                    {:path path}))]
-         (when ok-handler
-           (ok-handler repo path result)))
-       (fn [error]
-         (if error-handler
-           (error-handler error)
-           (log/error :delete-file-failed error))))))
+  (delete-file! [_this _repo _dir _path _opts]
```

Comment:
`delete-file!` here could be removed, it should be removed too from the protocol.